data_initialisation.py

```diff
@@ -250,6 +291,9 @@ def clean_regulatory_elements(regulatory_elements):
 
```

Comment:
Removed as any exception should be caught in read_regulatory_elements()

Reply:
You should still include an error condition in any function where they can emerge. You will find it harder to debug if not; and, in following this line of hoping, you end up in a sort of situation where you say, 'why are you so sure that read_regulatory_elements()' will catch it? Also, what if you had the same attitude towards that, and assumed that another function upstream of read_regulatory_elements() would catch the error instead? So it's safer to go with the original suggestion: it will make debugging easier and more helpful, and it does not make any assumption that other stuff will catch errors.